docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1289,12 +1292,32 @@ message PkgInfo {
   string url = 8;
   // Miscellaneous metadata, structured; a way to extend PkgInfo
   repeated StructuredAnnotation structured_annotations = 9;
+  // If set, specifies the properties that the underlying platform should have.
+  // If the platform does conform to these properties, the server should reject
+  // the P4Info when used with a SetForwardingPipelineConfigRequest.
+  PlatformProperties platform_properties = 11;
+}
+~ End Proto
+
+where the `PlatformProperties` message looks as follows:
+~ Begin Proto
+// Used to describe the required properties of the underlying platform.
+message PlatformProperties {
+  // The maximum number of multicast entries (i.e. multicast groups) that the
```

Comment:
maximum -> minimum
expected -> required
?

Reply:
Same a few more times.

Reply:
Good points. The `max_replicas_per_entry` is a bit tricky because, for the action profile equivalent, the server is required to reject anything greater than the `max_group_size`. I think this new wording works though.